src/Components/Cart/Cart.js

```diff
@@ -9,7 +9,7 @@ const Cart = () => {
   return (
     <div>
       {cartItems.map((cartItem) => (
-        <CartItem cartItem={cartItem} key={cartItem} />
```

Comment:
You may have noticed an error message saying \`"Warning: Each child in a list should have a unique "key" prop."\`

Whenever you iterate over items (eg: with `.map`) in the React JSX code (ie: inside a `render` function for class components, or inside the `return` value of a functionc omponent), you need to set a `key` value and that value has to be unique for each item.

Here you were setting the key to be the whole `cartItem`, but the probelm is that React expects the key to be a string. When it tries to convert `cartItem` to a string it will become `"[Object object]"`. And each `cartItem` will end up having that same `[Object object]` string as their key.

So instead, let's use the `id` which is generated in the cart slice reducer: [https://github.com/DYA13/Shop-online/blob/043985a3150fc2615b1f4404ec107b118748b05a/src/redux/cartSlice.js#L11](https://github.com/DYA13/Shop-online/blob/043985a3150fc2615b1f4404ec107b118748b05a/src/redux/cartSlice.js#L11/). That value is set to the time that the item was added, so it should likely be guaranteed to be unique for each item.